integrations/spark/openhouse-spark-runtime/src/main/scala/com/linkedin/openhouse/spark/sql/catalyst/plans/logical/SetReplicationPolicy.scala

```diff
@@ -0,0 +1,9 @@
+package com.linkedin.openhouse.spark.sql.catalyst.plans.logical
+
+import org.apache.spark.sql.catalyst.plans.logical.Command
+
+case class SetReplicationPolicy(tableName: Seq[String], replicationPolicies: Seq[(String, Option[String])]) extends Command {
```

Comment:
Should we consider setting up replication indefinitely or have default time upto which replication will be supported and if needed can be renewed?

Reply:
If we support indefinitely, replication would be done although some tables might not need. Or lets say user moved to new tables. In such case some clean up is required. Or the the whole process should be driven by users like how long they want to do the replication and if needed can be renewed. Let me also add this comment in the doc.

Reply:
this makes sense, @autumnust brought up a similar point where if replica tables are not frequently being read/accessed then we could stop the replication schedule. we would need a strategy to stop the replication schedule and cleanup the replication config on these types of replica tables.

Reply:
We can think about enforcing such policies in server side. I don't suppose that brings any change in SQL API spec.